scripts/sync-dashboards

```diff
@@ -0,0 +1,72 @@
+#!/usr/bin/env python3
+
+import os
+import sys
+import argparse
+from urllib.request import urlretrieve
+
+# sync the dashboards from https://opendev.org/openstack/sunbeam-charms/src/branch/main/charms/openstack-exporter-k8s/src/grafana_dashboards
+BASE_URL = "https://opendev.org/openstack/sunbeam-charms/raw/branch/main/charms/openstack-exporter-k8s/src/grafana_dashboards/"
```

Comment:
Can you join the `BASE_URL` and `DASHBOARDS` as `DASHBOARS_URLS` (or some other name), the `BASE_URL`  on its own is a 404

Reply:
Thanks